### ISSUE TITLE
Skipping low coverage genes

### ISSUE THREAD

**Issue by MichelledeGroot** [Illumina/ExpansionHunter#92]
Hi,

I noticed that genes with a coverage below 10 are skipped in the calling. However, when I used your other tool GraphAlignmentViewer, I noticed that these images sometimes show a correct repeat size. Thats why I wanted to ask whether it is possible to lower the threshold of 10 since some images show a proper repeat size while they are skipped by ExpansionHunter.

**Comment by egor-dolzhenko** [comment]
Yes, it makes sense that EH sometimes can correctly genotype repeats at low depths. We could consider reporting our best-guess genotype even when the depth is very low while setting the low-depth genotype filter. How urgently do you need this feature?

**Comment by MichelledeGroot** [comment]
It would be great if this feature could be added. It would be nice if it could be added within a month, but if it is not possible I could still use it later.
